### PR TITLE
chore: disable renovate update for swc

### PR DESCRIPTION
**Pull request by hardfist**
## Summary
swc may introduce breaking during patch update which is not safe to do auto bump for rspack, 
we need to do manually update for swc related crated

## Checklist

- [x] Tests updated (or not required).
- [x] Documentation updated (or not required).

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -67,7 +67,8 @@
       ],
       "matchPackageNames": [
         "styled_components"
-      ]
+      ],
+      "enable": false
```
Comment (chenjiahan) [inline .github/renovate.json:71]: how about use json5 ext and add some comments

Comment (chenjiahan) [reply .github/renovate.json:71]: see https://github.com/web-infra-dev/rsbuild/blob/main/.github/renovate.json5

Comment (hardfist) [reply .github/renovate.json:71]: json5 is better, I will change to json5

Comment (hardfist) [reply .github/renovate.json]: changed to json5 now

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**